i18n/en.json

```diff
@@ -2140,7 +2140,7 @@
   },
   {
     "id": "api.user.send_mfa_change_email.error",
-    "translation": ""
+    "translation": "Unable to send email notification for MFA change."
```

Comment:
Is this when the MFA config setting is updated, or when the user adds or removes MFA to their account?